README.md

```diff
@@ -13,7 +13,7 @@ There is also a collection of accessibility-related utility code, including but
 
 To include just the javascript rules, require the following file:
 
-    https://raw.github.com/GoogleChrome/accessibility-developer-tools/stable/dist/js/axs_testing.js
+    https://cdn.rawgit.com/GoogleChrome/accessibility-developer-tools/master/dist/js/axs_testing.js
```

Comment:
This content would be hosted by a third-party that is unaffiliated with Github[1]. That makes me uncomfortable and would require vetting the third-party. @alice what do you think?

[1]https://stackoverflow.com/questions/17341122/link-and-execute-external-javascript-file-hosted-on-github#comment30352321_18049842